test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -214,12 +214,12 @@ protected virtual void DeleteFolders()
                     f.Delete(true);
                 }
             }
-            catch (Exception e)
-            {
-                Console.WriteLine(e);
-            }
+            catch (Exception e)
```

Comment:
Mohsen, why do we have this fake change? You should understand that having a lot of fake changes could be a reason of PR closing.
What's your IDE settings?
- line-endings aka EOL is LF (Linux) or CRLF (Windows)?
- what's ISE overriding you have for style / formatting checks?

FYI We have mixed line endings. We allow any line ending for new added lines, but we reject any changes of EOL aka fake changes.
You must provide correct style & formatting settings for your Visual Studio IDE.
We have a special file which override and apply special formatting rules. But these rules work in Visual Studio only!

Reply:
with default settings and unchanged vs. Formatting is done by pressing ctrl + k, ctrl + d.
https://learn.microsoft.com/en-us/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2022#bkmk_edit-popular-shortcuts

Reply:
Understood! Again... Please refrain from refactoring, including style alterations. Any PRs should be free of any insubstantial changes. Endeavor to avoid introducing such changes.
Currently, I am examining the PR code and addressing these unnecessary changes before requesting an official review from the team... 😏

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/2125/commits/c2d7fb60563b1b832883314c3153721004065d49 
No fake changes now!